docs/database/beta.md

```diff
@@ -0,0 +1,105 @@
+# Database beta
+
+!!! warning
+    This product is in beta. This means there can be instabilities etc. We have stepped up our backup procedures on beta program, to have more often backups.
+    We never suggest you run production workloads during a beta.
+
+As our strive to always provide the best product to our clients, we are starting to roll out a beta of a completely new Database system. The idea is to make it more flexible and give better control over your database.
+
+## Core concept
+
+The basic idea is to setup a database server for an application. The system will automatically make sure there is a database for each environment.
+
+The database server is externally accessible, with the credentials provided through the API. This can be used to connect through Sequel Pro, CLI and other applications depending on your database engine.
+
+## Supported database engines and versions
+
+* MySQL
+    * 5.7.x (Default)
+    * 5.6.x
+    * 5.5.x
+* PostgreSQL
+    * 10.0.x (Default)
+    * 9.6.x
+    * 9.5.x
+    * 9.4.x
+    * 9.3.x
+* MongoDB
+    * 3.4.x (Default)
+    * 3.2.x
+    * 3.0.x
+
+## Commands
+
+You can see the commands under the `database` command.
+
+```
+vapor-beta cloud database --help
+```
+
+### Setting up database server
+
+```
+vapor-beta cloud database create --app=<your-app>
```

Comment:
i think we can omit `--app` here since we usually assume they will run the command from inside the project directory.

if not, we ask which application (numbered list)